docs/reference/crd-config/connnector-catalog.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Connector catalog CRD configurations
+category: reference
+id: connnector-catalog
+---
+
+The connector catelog CRD is only used to store metadata about built-in connectors in StreamNative Cloud. This table outlines the configurable fields of built-in connectors.
+
+## Built-in connectors in StreamNative Cloud
+
+| Field | Description |
+| ---|---|
+| `id` | The ID of the connector. |
+| `name` | The name of the connector type. |
+| `description` | The description that is used for user help. |
+| `sourceClass` | The class name for the connector source implementation. If not defined, it assumes that the connector cannot act as a data source. |
+| `sinkClass` | The class name for the connector sink implementation. If not defined, it assumes that the connector cannot act as a data sink. |
+| `sourceConfigClass` | The class name for the source configuration implementation. If not defined, the framework cannot do any submission time checks. |
+| `sinkConfigClass` | The class name for the sink configuration implementation. If not defined, the framework cannot do any submission time checks. |
+| `version` | The version of the connector. |
+| `imageRegistry` | The image registry that hosts the connector image. By default, the image registry is empty, which means that it uses the Docker Hub (`docker.io/`) to host the connector image. |
+| `imageRepository` | The image repository to the connector. Usually, it is in a format of NAMESPACE/REPOSITORY. |
+| `imageTag` | The tag to the connector image. By default, it aligns with Pulsar's version. |
+| `typeClassName` | The type class name of the connector or function. By default, it is set to `'[B'`.|
+| `sourceTypeClassName` | The type class name of the source connector. If not set, it will inherit the value from the `typeClassName` field. |
+| `sinkTypeClassName` |  The type class name of the sink connector. If not set, it will inherit the value from the `typeClassName` field. |
+| `jarFullName` | Optional. The JAR or NAR package name of the connector. If not set, it is generated based on the connector ID and connector version.| 
+| `defaultSchemaType` |  Optional. The default schema type of the connector's topic. |
+| `defaultSerdeClassName` | Optional. The default serde class name of the connector's topic. |
```

Comment:
I think this is missing some properties. It should contain all of these:
https://github.com/streamnative/function-mesh/blob/master/config/crd/bases/compute.functionmesh.io_connectorcatalogs.yaml#L33-L98

Reply:
@alpreu Thanks for your comments. Add missed properties. Could you please help look in to it when you have time? Thanks.